src/e2e/specs/purchase.spec.ts

```diff
@@ -72,18 +72,15 @@ test.describe(`${process.env.E2E_TEST_ENV} - Breach Scan, Monitor Plus Purchase
     // fill out subscription payment
     await purchasePage.authorizationCheckbox.check();
     await purchasePage.fillOutStripeCardInfo();
-    await purchasePage.payNowButton.click();
+    await purchasePage.payNowButton.click({ force: true });
```

Comment:
Q: Why does this need `{ force: true }`?
https://www.bondaracademy.com/blog/do-not-force-playwright

Reply:
I believe this is from the code I merged from rafee's [pr](https://github.com/mozilla/blurts-server/pull/4640#pullrequestreview-2105519406)

Reply:
I ran into a flaky test once where the button was clicked/selected but nothing happened, thought this might help 🤷